Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationFailureRepository.cs

```diff
@@ -83,6 +83,11 @@ public bool HasValidationFailures ()
         return _rowFailures is { Count: > 0 } || _cellFailures.Count > 0 && _cellFailures.Values.SelectMany(e => e).Any();
       }
 
+      public bool HasUnhandledValidationFailures ()
+      {
+        return _rowFailures.Any(f => !f.Handled) || _cellFailures.Values.Any(e => e.Any(f => !f.Handled));
```

Comment:
Readability question:
```suggestion
        return _rowFailures.Any(f => !f.Handled) || _cellFailures.Values.SelectMany(f => f).Any(f => !f.Handled));
```

Reply:
Personally I think Any -> Any is more understandable when reading it. It's basically just: Are there any `x` with any `y` inside?

While selectmany is more like: Give me all `x`, now do any `x` have `y` inside? 

Logically speaking I think any -> any is easier to understand. If one has read much selectmany code, it might be easier to understand it by default though.